test/test_eyeglass_plugin.js

```diff
@@ -101,7 +106,23 @@ describe("EyeglassCompiler", function () {
     return build(builder)
       .then(outputDir => {
         assertEqualDirs(outputDir, fixtureOutputDir("basicProject"));
-      });
+      })
+      .catch(assertFail);
+  });
+
+  it("compiles sass files with option.relativeAssets set", function () {
+    let optimizer = new EyeglassCompiler(fixtureSourceDir("basicProject"), {
+      cssDir: ".",
+      relativeAssets: true
+    });
+
+    let builder = new broccoli.Builder(optimizer);
+
+    return build(builder)
+      .then(outputDir => {
+        assertEqualDirs(outputDir, fixtureOutputDir("basicProject"));
+      })
+      .catch(assertFail);
```

Comment:
the test should already fail if the promise is rejected, thats how mocha's promise support works.

Reply:
TIL -- i'll update the tests again.

Reply:
Updated! Confirmed the 1 new test fails without the code change and passes with it.